### DISCUSSION TITLE
Discussion topics for Feb 28, 2022 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#4565] [Show and tell]
Discussion topics for Feb 28, 2022 Engineering Sync

**Comment by cwinters-usds** [comment]
One of the things the devops team has been working on is automating cross-environment database restores -- so we can use the production database in the test environment, for example. And the mechanics are just about done -- see [PR 4563](https://github.com/CDCgov/prime-reportstream/pull/4563).

However, we want to ensure we don't copy over configurations for live receivers such that they get our test data from the different environment -- that would be bad. What are the steps we need to take to disable or modify receivers so this doesn't happen?
